scenarios/telemetry/python/telemetry_consumer.py

```diff
@@ -0,0 +1,169 @@
+        connected_cond.notify_all()
+
+def on_subscribe(client, _userdata, mid, _granted_qos):
+    global subscribed_prop
+    print(f"Subscribe for message id {mid} acknowledged by MQTT broker")
+    # # In Paho CB thread.
+    with subscribed_cond:
+        subscribed_prop = True
+        subscribed_cond.notify_all()
+
+def on_message(_client, _userdata, message):
+    # # In Paho CB thread.
+    print(f"Received message on topic {message.topic} with payload {message.payload}")
+    msg = str(message.payload.decode("utf-8"))
+    messages.append(msg)
+
+def on_disconnect(_client, _userdata, rc):
+    print("Received disconnect with error='{}'".format(mqtt.error_string(rc)))
+    global connected_prop
+    # # In Paho CB thread.
+    with connected_cond:
+        connected_prop = False
+        connected_cond.notify_all()
+
+def wait_for_connected(timeout: float = None) -> bool:
+    with connected_cond:
+        connected_cond.wait_for(lambda: connected_prop or connection_error, timeout=timeout, )
+        if connection_error:
+            raise connection_error
+        return connected_prop
+
+def wait_for_subscribed(timeout: float = None) -> bool:
+    with subscribed_cond:
+        subscribed_cond.wait_for(
+            lambda: subscribed_prop, timeout=timeout,
+        )
+        return subscribed_prop
+
+def wait_for_disconnected(timeout: float = None):
+    with connected_cond:
+        connected_cond.wait_for(lambda: not connected_prop, timeout=timeout, )
+
+def create_mqtt_client(client_id, connection_settings):
+    mqtt_client = mqtt.Client(
+        client_id=client_id,
+        clean_session=connection_settings['MQTT_CLEAN_SESSION'],
+        protocol=mqtt.MQTTv311,
+        transport="tcp",
+    )
+    if 'MQTT_USERNAME' in connection_settings:
+        mqtt_client.username_pw_set(
+            username=connection_settings['MQTT_USERNAME'],
+            password=connection_settings['MQTT_PASSWORD'] if 'MQTT_PASSWORD' in connection_settings else None
+        )
+    if connection_settings['MQTT_USE_TLS']:
+        context = ssl.SSLContext(ssl.PROTOCOL_TLSv1_2)
+
+        if connection_settings['MQTT_CERT_FILE']:
+            context.load_cert_chain(
+                certfile=connection_settings['MQTT_CERT_FILE'],
+                keyfile=connection_settings['MQTT_KEY_FILE'],
+                password=connection_settings['MQTT_KEY_FILE_PASSWORD']
+                )
+        if "MQTT_CA_FILE" in connection_settings:
+            context.load_verify_locations(
+                cafile=connection_settings['MQTT_CA_FILE'],
+            )
+        else:
+            context.load_default_certs()
+
+        mqtt_client.tls_set_context(context)
+    return mqtt_client
+
+def main():
+    connection_settings = cs.get_connection_settings(args.env_file)
+    if not connection_settings["MQTT_CLEAN_SESSION"]:
+        raise ValueError("This sample does not support connecting with existing sessions")
```

Comment:
If we're printing this, can we reflect that we aren't supporting this yet in the env var readme? (I think this is currently true for all of the samples)